packages/voorbeeld-design-tokens/figma/voorbeeld.tokens.json

```diff
@@ -1815,9 +1815,9 @@
       }
     }
   },
-  "components/breadcrumb": {
+  "components/breadcrumb-navigation": {
     "todo": {
-      "breadcrumb": {
+      "breadcrumb-navigation": {
```

Comment:
Bij de componenten (in CSS class names en in tokens) gebruiken we bij Utrecht consistent `-nav`, dus ook `utrecht-breadcrumb-nav`. Dat is dan zoals het `<nav>` element in HTML. Dat geldt ook voor de react code zoals `<BreadcrumbNav />` en de web component (`<utrecht-breadcrumb-nav>`).

Zullen we ook hier `breadcrumb-nav` gebruiken?

Zeker voor langere onderdelen, zoals `breadcrumb-nav-separator` blijft het dan iets overzichtelijker.

Reply:
Ok, dus bedoel je dan class name, design tokens etc. met `-nav` en de component name met `navigation`?

Ook voor `side-nav`, `form-nav` etc.?

Ik vraag me dan namelijk af of het niet meer voorspelbaar is als de naam van het component overeenkomt met deze class name en token name...

Misschien moeten we dan toch overwegen om, tegen onze eerdere afspraak in de componenten die `navigation` zijn `nav` te noemen 🫣

Reply:
Yes, Sidenav bij is bij Den Haag ook al afgekort, maar niet als los onderdeel. Dus `Sidenav` ipv `SideNav` en `denhaag-sidenav` in plaats van `denhaag-side-nav`

https://github.com/nl-design-system/denhaag/blob/main/components/Sidenav/src/Sidenav.tsx